morebits.js

```diff
@@ -5868,6 +5861,9 @@ Morebits.simpleWindow = function SimpleWindow(width, height) {
 
 	// resize the scrollbox with the dialog, if one is present
 	$widget.resizable('option', 'alsoResize', '#' + this.content.id + ' .morebits-scrollbox, #' + this.content.id);
+
+	// add skin-invert to "close" button
+	$('.morebits-dialog .ui-dialog-titlebar-close').addClass('skin-invert');
```

Comment:
This feels like a hack, but looks like there's no other way to do it - as the 'x' is an icon rather than text.

Reply:
I agree it's a hack. Long term we should probably move away from jQuery to ES6.